lib/internal/Magento/Framework/App/FeedFormatsInterface.php

```diff
@@ -0,0 +1,11 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Framework\App;
+
+interface FeedFormatsInterface
+{
+    const DEFAULT_FORMAT = 'rss';
```

Comment:
I think this approach would violate SRP as feed formats and output formats are different things.